tests/orgs/editOrg/endpoint.test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var test = require('tape');
+var init = require('../../../example/server.js');
+var config = require('../../../config/load-config.js');
+
+// test endpoint
+test('successful update returns an empty array', function (t) {
+  init(config, function (err, server, pool) {
+    var orgObj = {
+      name: 'Google',
+      logo_url: 'www.google.co',
+      mission_statement: 'Change the internet'
+    };
+
+    server.inject({
+      method: 'POST',
+      url: '/updateOrg?id=1',
+      payload: orgObj
+    }, function (res) {
+      t.deepEqual(res.result, [], 'successful update returns an empty array');
+      t.end();
+      pool.end();
+      server.stop();
+    });
+  });
+});
+
+
+// test endpoint
+test('update to org that doesnt exist ', function (t) {
+  init(config, function (err, server, pool) {
+    var orgObj = {
+      name: 'Google',
+      logo_url: 'www.google.co',
+      mission_statement: 'Change the internet'
+    };
+
+    server.inject({
+      method: 'POST',
+      url: '/updateOrg?id=10000',
+      payload: orgObj
+    }, function (res) {
+
+      // t.equal(res.statusCode, 404, 'Bad org id returns 404 error');
+
```

Comment:
Can we delete this test and just add the endpoint to a list on an issue where we need to catch for errors concerning tags/orgs/challenges that don't exist? Only because it's not testing anything at all atm...